docs/source/getting-started.mdx

```diff
@@ -153,8 +153,8 @@ Open `index.ts` in your preferred code editor and paste the following into it:
 <MultiCodeBlock>
 
 ```ts title="index.ts"
-import { ApolloServer } from '@apollo/server'; // preserve-line
-import { startStandaloneServer } from '@apollo/server/standalone'; // preserve-line
+import { ApolloServer } from '@apollo/server';
```

Comment:
@shorgi I believe this was part of making things look good when converted to JS. It looks like this code block doesn't support JS at all any more. Was this an intentional change — does our doc system no longer support the multi code block?  I'm not sure how I feel about that change if so....... but in any case, if we no longer support the automatic JS transpilation, there's a NOTE a few paragraphs up about JS that needs to be updated.

Reply:
@glasser the new docs do still support MultiCodeBlock (see the [page in new docs](https://www.apollographql.com/docs/apollo-server/getting-started#step-3-define-your-graphql-schema)) 

But since the new docs no longer do TS -> JS automatically, the `preserve-line` comment is no longer necessary, so they all can be removed (and not published in docs).

I'm unsure which NOTE you're referring to that needs updating, can you point it out?

Reply:
Hmm, I didn't realize this had gone either, but it appears that we no [longer do typescript -> JavaScript automatically](https://apollograph.slack.com/archives/C0721M2F6/p1729692264427289?thread_ts=1729643320.828069&cid=C0721M2F6).

Daniel's OOO, but will see if there's a non-automatic way to support this currently or if it needs to be added back in.